README.md

```diff
@@ -1 +1,189 @@
+
+Creating your first "Hello-microverse" project
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+-A Git hub account
+<br>
+-Node JS
+<br>
+-Visual Studio Code or any other code editor of your preference
+<br>
+-web browser
+
+### Setup
+
+Clone this repository to your desired folder:<br>
+Advantages of Linters:<br>
+
+1: improves readability-it will be easy to read and make adjustment if required easily.<br>
+2:removes silly errors before execution and code review.<br>
+3:Makes code look like written by a single person.<br>
+4:Spreads awareness and ownership over code quality<br>
+5:Having more secure and performant code.<br>
+
+## Install Linters
+
+You can find linters for most of the programming languages, e.g. Rubocop for Ruby or ESLint for JavaScript.
+
+Also, there are many ways you can integrate a linter in your workflow:
+
+-text editor plugin<br>
+-GitHub Actions<br>
+-GitHub apps<br>
+
+## Set up Linters
+
+**Note:** The npm package manager is going to create a node_modules directory to install all of your dependencies. You shouldn't commit that directory. To avoid that, you can create a .gitignore file and add node_modules to it:
+
+# .gitignore
+
+node_modules/
+
+## Web Hint
+
+This is a customizable linting tool that helps you improve your site's accessibility, speed, cross-browser compatibility, and more by checking your code for best practices and common errors.
+
+**NOTE:** If you are using Windows, make sure you initialize npm to create `package.json` file.
+
+```
+npm init -y
+```
+
+1. Run
+   ```
+   npm install --save-dev hint@7.x
+   ```
+   _how to use npm: (https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)._
+2. Copy [.hintrc](.hintrc) to the root directory of your project.
+3. **Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students.**
+   - If you think that change is necessary - open a [Pull Request in this repository](../README.md#contributing) and let your code reviewer know about it.
+4. Run
+   ```
+   npx hint .
+   ```
+5. Fix validation errors.
+
+### [Stylelint](https://stylelint.io/)
+
+A mighty, modern linter that helps you avoid errors and enforce conventions in your styles.
+
+1. Run
+
+npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+not sure how to use npm? Read this.
+
+2. Copy .stylelintrc.json to the root directory of your project.
+
+3. **Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students.**
+
+If you think that change is necessary - open a Pull Request in this repository and let your code reviewer know about it. 4. Run npx stylelint "\*_/_.{css,scss}" on the root of your directory of your project.
```

Comment:
- [ ] [OPTIONAL] There is no need to guide the repository users on how the linter checkers are installed. Kindly consider removing the steps used in linter installation to make your README more professional ☺️.